Release/tests/functional/http/client/outside_tests.cpp

```diff
@@ -63,6 +63,27 @@ TEST_FIXTURE(uri_address, outside_cnn_dot_com)
     });
 }
 
+TEST_FIXTURE(uri_address, outside_wikipedia_compressed_http_response)
```

Comment:
Thank you, this is a much needed change. We've had a longstanding issue with this test failing outside the US.